source/MQTTFileDownloader.c

```diff
@@ -335,30 +375,156 @@ static MQTTFileDownloaderStatus_t handleCborMessage( uint8_t * decodedData,
     return handleStatus;
 }
 
-static MQTTFileDownloaderStatus_t handleJsonMessage( uint8_t * decodedData,
-                                                     size_t * decodedDataLength,
-                                                     uint8_t * message,
-                                                     size_t messageLength )
+static bool getNumberFromString( char * str,
```

Comment:
Can this function convert a negative number from a string to an `int32_t`? I don't see any logic for handling a negative sign in it

Reply:
No, it cannot. I added that to the brief.

Reply:
If it can't handle a negative number is there a reason it returns an `int32_t`instead of a `uint32_t`? Wouldn't it allow for a larger number to be returned too then?